renovate.json

```diff
@@ -32,6 +32,10 @@
       {
         "matchPackageNames": ["Django"],
         "allowedVersions": "=3.2"
+      },
+      {
+        "matchPackageNames": ["wagtail"],
+        "allowedVersions": "< 4.2"
```

Comment:
Yeah! But can we go for one version? So, either here "=4.1" or above also "<3.3"

Reply:
I initially went for `=4.1` but then was unsure since we are still on `4.0` and wouldn't get updates for that.

Reply:
Oh sorry, only saw that now. Yes, that makes sense. Let's change both!